README.md

```
@@ -1,7 +1,24 @@
-## 👾 덕후들이 사랑하는 거래공간 : 납작마켓 👾
-<img src="https://github.com/user-attachments/assets/003b8003-2fbf-4c77-a9a7-aede81e4a123" width = "800"/>
+<img width="100" src="https://github.com/user-attachments/assets/ee232d34-10e4-407e-89f3-d3d6306a6aaf" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)